### ISSUE TITLE
[Request] Different default memo composition for tasks vs lists

### ISSUE THREAD

**Issue by pdxrlk** [Quorafind/Obsidian-Memos#124]
For tasks, I only want **{CONTENT}** but for lists I want **{TIME} {CONTENT}**

Thanks for considering this - love the plug-in - thank you.

**Comment by Quorafind** [comment]
The question is that when using only CONTENT, the memos cannot know when the memo is created(because HH:mm is used for recognized as time part of it. Or I have an idea, if I make all the tasks with content be the first part of memos created in that day(always) like:

- 02:00 list one
- [ ] task one

and the task one always on top of it in memos view. Do you think this is a good idea?

**Comment by pdxrlk** [comment]
Yes I think that's a good idea.  I would prefer putting the tasks first, instead of last (otherwise it is easy to overlook them).  Maybe a preference setting ( tasks first, tasks last) ?

**Comment by Quorafind** [comment]
sure, this would have a option to open this feature and toggle tasks first or last

**Comment by Quorafind** [comment]
By the way, the new feature that shows the TASK label near time text should help you something.
